lib/addons/konveyor/tackle-manifest.ytpl

```diff
@@ -0,0 +1,39 @@
+kind: Tackle
+apiVersion: tackle.konveyor.io/v1alpha1
```

Comment:
v1alpha1 makes me think this product is not production ready.

Reply:
Hello,

I'm a maintainer of Konveyor and can provide some more context of Konveyor's status.
Konveyor is an actively developed and maintained CNCF sandbox project that is the upstream for [Red Hat's Migration Toolkit for Applications ](https://developers.redhat.com/products/mta/overview).   

The code is in production at multiple large organizations being used to aid modernization engagements of enterprise java applications to Kubernetes.  The code that makes up Konveyor began ~10 years ago on the Analysis side with the latest version of the platform being developed ~2 years ago, the solution was called Tackle2.  

After being accepted into CNCF,  there has been an effort to refocus Konveyor onto a [Unified vision](https://github.com/konveyor/enhancements/tree/master/enhancements/unified_experience) of bringing the functionality that used to be in separate projects together into a common solution which is Konveyor.  

As part of this refocus, we decided to release Konveyor as a "v0.1" as opposed to continuing with the Tackle 2.1+ scheme that was in place prior.  

It is possible for the API surface to change as we are working with the community to shape what the future will be.  In regard to the API surface for the operator CR, we are careful to keep upgrades working and stable and have automated tests and QE resources actively helping to ensure stability.

Reply:
Hello @jwmatthews, thank you for your feedback, it surely helps. Understand the comment related to the CR API version and upgrades. It is not as intuitive but it surely matches the saying "there is nothing more permanent than temporary" when it comes versioning. 

I assume we should change the source references to https://github.com/konveyor/tackle2-operator, please confirm. 

Is https://konveyor.github.io/ still the right documentation entry point? I saw a reference to tackle document [here](https://github.com/konveyor/tackle2-operator) pointing to what appears to be the right page: https://konveyor.github.io/tackle. This url, however, is not functioning. 

Understand that doc maintenance may not be the top priority, I was a lot more interesting in dev activities, which you confirmed. Thank you.

Reply:
Yes, https://konveyor.github.io/ is the right documentation entry point, we are bringing back usage of this as the central spot for developers to contribute to.

The source code for the Konveyor operator is located at: https://github.com/konveyor/tackle2-operator

If you want to reference source code, use: https://github.com/konveyor/tackle2-operator
If you want to reference documentation use: https://konveyor.github.io/ 

As to https://konveyor.github.io/tackle, that is an older link and not intended.